### PR TITLE
Update generic reports

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/19365

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Generics_Footer_Report.sql [hunk 1]
```diff
@@ -0,0 +1,58 @@
+DROP FUNCTION IF EXISTS de_metas_endcustomer_fresh_reports.Docs_Generics_Footer_Report(IN p_org_id numeric)
+;
```
Comment (cristinamghita) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Generics_Footer_Report.sql:2]: If you introduce this function, then please used it in ```jasperreports/de/metas/docs/generics/report_footer.jrxml```, in metasfresh repo.

**Review by cristinamghita — CHANGES_REQUESTED** [review]
see my comment

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Generics_Footer_Report.sql (hunk 1 above)
Comment (riadmen) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Generics_Footer_Report.sql:2]: done

**Review by cristinamghita — APPROVED** [review]
lgtm

**Pull request merged**